server/jamovi/server/instance.py

```diff
@@ -743,6 +746,7 @@ def _on_dataset_ins_cols(self, request, response):
                 column.active = column_pb.active
                 column.filter_no = column_pb.filterNo
                 column.trim_levels = column_pb.trimLevels
+                column.transform = column_pb.transform
```

Comment:
here's your problem. `column_pb.transform` isn't a list. it's some sort of protobuf object. change this to:

```
column.transform = list(column_pb.transform)
```

and you should be in business

Reply:
calling `list(iterable)` creates a list from the iterable object. it's common to use `list(map(...))` too for example

Reply:
Yep that was the problem i was having but not on this line. This line is fine. It was with the transform BP not the column PB. I have committed the fix.